### ISSUE TITLE
Package.activateConfig is deprecated.

### ISSUE THREAD

**Issue by nicolasmccurdy** [yujinakayama/atom-auto-update-packages#27]
Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details
```
Package.activateConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:263:11)
Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:234:14)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:219:30)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:163:15)
Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:211:14)
PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:435:21)
```

**Comment by iamstarkov** [comment]
:+1:

**Comment by iamstarkov** [comment]
@yujinakayama Hi, I’m using your plugin and found it helpful and useful, that’s why I want it to be updated before it will be broken when Atom@1.0 will be released in june

**Comment by tdbs** [comment]
:+1:

**Comment by yujinakayama** [comment]
Fixed in auto-update-packages 1.0.0. (#31)
